dbms/src/Storages/Page/V3/BlobStore.cpp

```diff
@@ -561,6 +598,14 @@ Page BlobStore::read(const PageIDAndEntryV3 & id_entry, const ReadLimiterPtr & r
     const auto & [page_id_v3, entry] = id_entry;
     const size_t buf_size = entry.size;
 
+    if (buf_size == 0)
+    {
+        LOG_FMT_DEBUG(log, "Read entry [page_id={}] without entry size.", page_id_v3);
+        Page page;
+        page.page_id = page_id_v3.low;
+        return page;
+    }
```

Comment:
Maybe we should also make the default cons of `ByteBuffer` to init its `begin_pos` and `end_pos` to be nullptr